examples/gce-vpn/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "project" {
```

Comment:
:+1: for example! This will help users to get up to speed, it would be even more helpful if you add a brief Readme describing the setup, similar to the ones we already have for each example.
https://github.com/hashicorp/terraform/tree/master/examples